downloads/index.md

```diff
@@ -8,14 +8,14 @@ title:  Julia Downloads
 We provide several ways for you to run Julia:
 
 * In the terminal using the built-in Julia command line.
-* The [Juno](http://www.junolab.org) integrated development environment (IDE).
 * In the browser on [JuliaBox.com](https://www.juliabox.com) with Jupyter notebooks. No installation is required -- just point your browser there, login and start computing.
+* The [JuliaPro distribution](http://juliacomputing.com/products/juliapro.html) by Julia Computing provides a single installer for Julia, the [Juno IDE](http://junolab.org), the Gallium debugger, and a number of packages (requires registration).
 
 Plotting capabilities are provided by external packages such as
-[PyPlot.jl](https://github.com/stevengj/PyPlot.jl) and [Gadfly.jl](http://gadflyjl.org). 
-A package which integrates most of Julia's plotting backends into one convenient and 
-well-documented API is [Plots.jl](https://github.com/tbreloff/Plots.jl). Look at the 
-[plotting instructions](plotting.html) to install a plotting package. If you are using 
+[PyPlot.jl](https://github.com/stevengj/PyPlot.jl) and [Gadfly.jl](http://gadflyjl.org).
```

Comment:
While you're in here it might be nice to update the URL for PyPlot, as it's now in the JuliaPy organization.